### PR TITLE
README: Improve formatting and capitalization.

### PR DESCRIPTION
**Pull request by johnjohndoe**
- Enable code formatting for commands.
- Enable syntax highlighting.
- Capitalize product and proper names.
- Preview here: https://github.com/johnjohndoe/riot-android/blob/patch-1/README.md

### CODE REVIEW

**Comment by johnjohndoe** [comment]
:heavy_check_mark: Rebased and conflicts resolved.

File: CHANGES.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ Improvements 🙌:
  - Remove jQuery library. Fixes #3381
 
 Other changes:
+ - Fix F-Droid badge and improve formatting and capitalization in README (#3492)
```
Comment (bmarty) [inline CHANGES.rst:15]: What is the fix on F-Droid badge?

Comment (bmarty) [reply CHANGES.rst:15]: How I think it has vanished after the rebase.

Comment (johnjohndoe) [reply CHANGES.rst:15]: Yes, it disappeared with the rebase. The sentence part can be removed from CHANGES.rst.

**Pull request merged**